holidays/countries/india.py

```
@@ -256,22 +292,41 @@ def _populate_optional_holidays(self):
 
-        if self.subdiv != "MH":
-            # Holi.
-            self._add_holi(tr("Holi"))
-
         # Ganesh Chaturthi.
         self._add_ganesh_chaturthi(tr("Ganesh Chaturthi"))
 
         # Govardhan Puja.
         self._add_govardhan_puja(tr("Govardhan Puja"))
 
+        if self._year == 2008 or self._year == 2017:
```

Ruff (0.15.12)

[warning] Consider merging multiple comparisons: `self._year in {2008, 2017}`.

Merge multiple comparisons

(PLR1714)